lib/matplotlib/cm.py

```diff
@@ -92,10 +92,10 @@ def __init__(self, cmaps):
         self._builtin_cmaps = tuple(cmaps)
 
     def __getitem__(self, item):
-        try:
-            return self._cmaps[item].copy()
-        except KeyError:
-            raise KeyError(f"{item!r} is not a known colormap name") from None
+        cmap = _api.check_getitem(
+            self._cmaps, colormap=item, _error_cls=KeyError
+        )
```

Comment:
This fits in a line.
```suggestion
        cmap = _api.check_getitem(self._cmaps, colormap=item, _error_cls=KeyError)
```

Reply:
And if you are talking about this as style, `black` would have put it on one line.

Reply:
My objection is mainly to bikeshedding style one comment at a time in PR review, instead of having a tool automate it for us.